packages/destination-actions/src/destinations/adobe-target/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import updateProfile from './updateProfile'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Adobe Target Cloud Mode',
+  slug: 'actions-adobe-target-cloud',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      client_code: {
+        label: 'Client Code',
+        description:
+          'Your Adobe Target client code. To find your client code in the Target UI click Administration > Implementation. The client code is shown up top under Account Details.',
```

Comment:
Slightly changing this to match what I suggest for the web integration!

```suggestion
          'Your Adobe Target client code. To find your client code in Adobe Target, navigate to **Administration > Implementation**. The client code is shown at the top under Account Details.',
```